daemon/container_unix.go

```diff
@@ -1124,14 +1135,6 @@ func (container *Container) releaseNetwork() {
 		return
 	}
 
-	for _, ns := range networks {
-		n, err := container.daemon.FindNetwork(ns)
-		if err != nil {
-			continue
-		}
-		container.disconnectFromNetwork(n, false)
```

Comment:
Now that `disconnectFromNetwork` is removed from the releseNetwork, can you also make the correction to the
disconnectFromNetwork definition by removing the `updateSettings` parameter altogether ?
We use in only 2 places (one is here) and the other from the actual user triggered DisconnectFromNetwork.
And hence making the suggested changes will make the code read better.